pkgs/development/python-modules/uarray/default.nix

```
@@ -0,0 +1,44 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, matchpy
+, numpy
+, astunparse
+, typing-extensions
+, black
+, pytest
+, pytestcov
+, numba
+, nbval
+, python
+, isPy37
+}:
+
+buildPythonPackage rec {
+  pname = "uarray";
+  version = "0.4";
+  format = "flit";
+  # will have support soon see
+  # https://github.com/Quansight-Labs/uarray/pull/64
+  disabled = isPy37;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "4ec88f477d803a914d58fdf83aeedfb1986305355775cf55525348c62cce9aa4";
+  };
+
+  checkInputs = [ pytest nbval pytestcov numba ];
+  propagatedBuildInputs = [ matchpy numpy astunparse typing-extensions black ];
+
+  checkPhase = ''
+    ${python.interpreter} extract_readme_tests.py
+    pytest
```

```suggestion
    py.test
```
Seems to be their suggestion https://github.com/Quansight-Labs/uarray#testing

---

https://stackoverflow.com/questions/39495429/py-test-vs-pytest-command

`pytest` is the new way to invoke the command `py.test` since 3.0